metron-streaming/pom.xml

```diff
@@ -80,6 +80,7 @@
 		<module>Metron-Elasticsearch</module>
 		<module>Metron-Solr</module>
 		<module>Metron-Testing</module>
+		<module>Metron-TestingUtilities</module>
```

Comment:
Testing contains our integration test classes and depends on pretty much every project (Common, Parsers, etc).  TestingUtilities is meant to contain testing utilities (like MockHTable) that any project can leverage.